app/sidekiq/decision_review/submit_upload.rb

```diff
@@ -1,73 +1,100 @@
 # frozen_string_literal: true
 
 require 'decision_review_v1/service'
+require 'decision_review_v1/utilities/logging_utils'
 
 module DecisionReview
   class SubmitUpload
     include Sidekiq::Job
+    include DecisionReviewV1::Appeals::LoggingUtils
     STATSD_KEY_PREFIX = 'worker.decision_review.submit_upload'
 
     sidekiq_options retry: 5
 
+    sidekiq_retries_exhausted do |_msg, _ex|
+      StatsD.increment("#{STATSD_KEY_PREFIX}.permanent_error")
+    end
+
     # Make a request to lighthosue to get the URL where we can upload the file,
     # then get the file from S3 and send it to lighthouse
     #
     # @param appeal_submission_upload_id [String] UUID in response from Lighthouse upload
     # @param type [Symbol|String] type of submission one of 'SC','NOD' or :SC, :NOD (case insensitive) default: NOD
-
-    sidekiq_retries_exhausted do |_msg, _ex|
-      StatsD.increment("#{STATSD_KEY_PREFIX}.error")
-    end
-
+    # rubocop:disable Metrics/MethodLength
     def perform(appeal_submission_upload_id)
       appeal_submission_upload = AppealSubmissionUpload.find(appeal_submission_upload_id)
       appeal_submission = appeal_submission_upload.appeal_submission
-      sanitized_file = DecisionReviewEvidenceAttachment.find_by(
-        guid: appeal_submission_upload.decision_review_evidence_attachment_guid
-      )&.get_file
+      sanitized_file = get_sanitized_file(appeal_submission_upload)
       file_number_or_ssn = JSON.parse(appeal_submission.upload_metadata)['fileNumber']
+
       lh_upload_id = case appeal_submission.type_of_appeal
                      when 'NOD'
-                       handle_notice_of_disagreement(appeal_submission, file_number_or_ssn, sanitized_file)
+                       handle_notice_of_disagreement(appeal_submission_upload, file_number_or_ssn, sanitized_file)
                      when 'SC'
                        handle_supplemental_claim(appeal_submission, file_number_or_ssn, sanitized_file)
                      else
                        raise "Unknown appeal type (#{type})"
                      end.body.dig('data', 'id')
       appeal_submission_upload.update!(lighthouse_upload_id: lh_upload_id)
-      log_success(internal_id: appeal_submission.id, lighthouse_uuid: appeal_submission.submitted_appeal_uuid,
-                  lighthouse_evidence_uuid: lh_upload_id, appeal_type: appeal_submission.type_of_appeal)
       StatsD.increment("#{STATSD_KEY_PREFIX}.success")
+    rescue => e
+      log_formatted(
+        key: :evidence_upload_to_lighthouse,
+        form_id: '10182',
+        user_uuid: appeal_submission&.user_uuid,
+        upstream_system: 'AWS S3 or Lighthouse',
+        is_success: false,
+        body: e.message, # Test 504 error as an example
```

Comment:
From what I've seen, errors only occur when LH returns HTML 504s, instead of as JSON, in which case, I'd like to log the error message.